de-lib/src/main/java/org/iplantc/de/pipelineBuilder/client/json/App.java

```diff
@@ -24,6 +24,7 @@
     private List<PipelineAppData> outputAppData;
 
 	private int publicId;
+	private String systemId;
 	private String ID;
     private String templateId;
```

Comment:
Looks like several lines were indented with tabs. I reformatted the code using IntelliJ Idea. That should fix it.